WebServer/microservices/video-deleter/deleter.py

```diff
@@ -42,9 +42,9 @@ def periodically_delete(delay,days):
 
             #In the database
             response = requests.delete(dbr+"video/delete",headers={'Content-type': 'application/json'},json={"video_id":v_id})
-            # TODO: fix....
+
             if response.status_code != 200:
-                print(str(response.status_code) +": "+response.content.decode('utf-8'))
+                logger.error(f"Could not delete from database: {response.status_code} {response.content.decode('utf-8')}. video: {v_id}")
```

Comment:
Fint med logging. Måske det også skulle gøres i linje 40, hvis stien ikke eksisterer

Reply:
Jeg laver et commit med linje 40 også. 
Kan ikke svare direkte på din anden kommentar så: Søgte i alle filer efter "TODO" og det var det. Der er to TODOs mere inde i frontenden, men dem tager Casper sig af, da de ligger op ad det han er igang med i forvejen.